### PR TITLE
fix(TBD-11560): fix for lightweight classpath

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java [hunk 1]
```diff
@@ -67,6 +67,20 @@
             "$HADOOP_YARN_HOME/share/hadoop/yarn/*",
             "$HADOOP_YARN_HOME/share/hadoop/yarn/lib/*"
     ));
+    
+    public final static String DEFAULT_LIB_ROOT = "/opt/cloudera/parcels/CDH/lib";
```
Comment (amedvedenko) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: I still don't understand why is it hardcoded. Could you please confirm that this path will be good for all CDH distributions for all types of installation? For manual installation, like dkr does, Cloudera Manager installation or if this path is changed (if it's possible, I'm not sure).

Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: this is the default path for installations of any CDH / CDP clusters, when you use parcels from cloudera
if the path is changed, its up to the customer to use the right paths with the use custom classpath field

Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: _You can install parcels at any location in the filesystem. They are installed by default in /opt/cloudera/parcels. In contrast, packages are installed in /usr/lib._
ok, but is there a chance that there is no Parcels?

Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: then its a custom class path and user can point to any folder he'd like

Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: ok, is it specified in our docs that default value for application classpath with lightweight selected depends on default parcels value?

Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: the default value is specified and its said that if customer does not comply with those paths he should changed them ...

Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: specified in the Talend docs, rigth?

**Review by amedvedenko — APPROVED** [review]

File: main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java (hunk 1 above)
Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:71]: yes, Anne made an official document from the wiki

**Review by talend-jphautin — APPROVED** [review]

**Pull request merged**